kibana-reports/server/model/index.ts

```diff
@@ -143,13 +142,12 @@ export const channelSchema = schema.object({
     }),
     { minSize: 1 }
   ),
+  // TODO: consider add this field next to url-related fields.
+  // Need this to build the links in email
+  origin: schema.uri(), //e.g. https://xxxxx.com
```

Comment:
Do we need input validation for this?

Reply:
the `.uri()` will check whether it is valid uri in this format `https://xxxxx.com`